### PR TITLE
Add - Doc for input

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**

### CODE REVIEW

**Comment by aayushjain-code** [comment]
Nice components try to make it center align.

File: index.html [hunk 1]
```diff
@@ -536,6 +544,11 @@ <h1 class="mg-1-top mg-point6-bot title-lg hover-underline-animation">Label Inpu
                 <div class="success-msg">Password matched.</div>
             </div>
         </div>
+
+        <!-- Code Snippet -->
+        <iframe src="https://www.thiscodeworks.com/embed/61fb31896ab14d0015c32cbe" style="width: 100%; height: 333px;"
```
Comment (aayushjain-code) [inline index.html:549]: Try to use rem values

**Review by aayushjain-code — COMMENTED** [review]
LGTM

File: index.html [hunk 2]
```diff
@@ -511,6 +514,11 @@ <h1 class="mg-1-top mg-point6-bot title-lg hover-underline-animation">TextArea I
...
         </div>
 
+        <!-- Code Snippet -->
+        <iframe src="https://www.thiscodeworks.com/embed/61fb307c6ab14d0015c32cbd" style="width: 100%; height: 99px;"
+        frameborder="0"></iframe>
+        
+
         <h1 class="mg-1-top mg-point6-bot title-lg hover-underline-animation">Label Input</h1>
         <div class="mg-1-bot">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nam iusto, dicta harum sequi fugit
         deserunt?
```
Comment (vaibhav18matere) [inline index.html:524]: use descriptive classnames

**Pull request merged**